README.md

```diff
@@ -106,12 +116,21 @@ These are as follows:
 - AUTHORISED_ADMIN_GROUP: can do any required action
 - AUTHORISED_MANAGER_GROUP: same as officers, plus can assign applications to officers for assessment and see sensitive data
 - AUTHORISED_OFFICER_GROUP: can view applications and perform assessments on assigned applications
+- AUTHORISED_READONLY_GROUP: can view applications
 
 The scope and expectations around permissions have been kept fairly limited at this stage, but is an area for future enhancements.
 
 ### Pre-commit hooks
 
-Repository has pre-commit hooks configuration to prevent direct commits to main branches and for scanning secrets. Please ensure you have [pre-commit framework](https://pre-commit.com/) installed before starting development work.
+Repository has a husky configuration to prevent staged files commits that fail linting, test suites and for scanning secrets. On commit it will run linting on staged files, jest and cypress components tests. Pre-push it will run cypress e2e tests.
+
+### E2E tests
+
+A suite of e2e tests have been written with cypress. Check the env vars are set correctly for AUTHORISED\_\* groups before running tests.
+
+Standard e2e tests use nock to intercept network requests and mock responses. The configuration will start the application from within the cypress.config.ts to allow nock to work within the next environment sucessfully. These will also run in the pipeline and record video of failed tests.
+
+Local e2e tests in the local folder require all nock configuration to be commented out in cypress.config.ts, a local backend to be running and the LOCAL_E2E to be set to true.
```

Comment:
worth mentioning here that the local e2e tests should be ideally run against prod build to ensure they work as expected